python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -42,26 +40,25 @@ def _get_upstream_mapping(
 
 
 def _get_parent_updates(
-    context,
     current_asset: AssetKey,
     parent_assets: Set[AssetKey],
-    cursor_tuple: Tuple[float, int],
+    cursor: Optional[int],
     will_materialize_set: Set[AssetKey],
     wait_for_in_progress_runs: bool,
-    planned_materialization_cache: Dict[AssetKey, "EventLogRecord"],
-) -> Tuple[Mapping[AssetKey, Tuple[bool, Tuple[float, int]]], Dict[AssetKey, "EventLogRecord"],]:
+    instance_queryer: "CachingInstanceQueryer",
+) -> Mapping[AssetKey, Tuple[bool, Optional[int]]]:
     """The bulk of the logic in the sensor is in this function. At the end of the function we return a
     dictionary that maps each asset to a Tuple. The Tuple contains a boolean, indicating if the asset
-    has materialized or will materialize, and a tuple(float, int) representing the timestamp and storage id
+    has materialized or will materialize, and a storage ID
     the parent asset would update the cursor to if it is the most recent materialization of a parent asset.
     In some cases we set the tuple to (0.0, 0) so that the tuples of other parent materializations will take precedent.
 
     Args:
         current_asset: We want to determine if this asset should materialize, so we gather information about
             if its parents have materialized.
         parent_assets: the parents of current_asset.
-        cursor_tuple: In the cursor for the sensor we store the timestamp and storage id of the most recent materialization
-            of current_asset's parents. This allows us to see if any of the parents have been materialized
+        cursor: In the cursor for the sensor, we store the timestamp and storage id of the most recent materialization
```

Comment:
"we store the timestamp and storage id" isn't true anymore because `cursor` is an int right?